packages/pn-pa-webapp/src/pages/ApiKeys.page.tsx

```diff
@@ -85,14 +95,32 @@ const ApiKeys = () => {
   const { t } = useTranslation(['apikeys']);
 
   const apiKeys = useAppSelector((state: RootState) => state.apiKeysState.apiKeys);
+  const pagination = useAppSelector((state: RootState) => state.apiKeysState.pagination);
+
+  const totalElements =
```

Comment:
This is the pagination management for a differente structure. In this case we don't have moreResult or nextPagesKey.

So 
- totalElement = total
- pagesToShow = Math.min(pages, 3) where pages = Math.ceil(total / size)

Reply:
https://github.com/pagopa/pn-frontend/pull/926/commits/c7ee0db13db237837b18444852479d492db22fc6